python/simulators/simulators.cpp

```diff
@@ -226,6 +233,16 @@ class BlackOilSimulator
         delete[] argv_;
     }
 
+    void setupCmdLineArgs()
```

Comment:
I do not understand what this function is needed/used for. Maybe it can be removed entirely?